tests/system/src/test/java/test/javafx/accessibility/virtualflow/VirtualFlowMemoryLeakTest.java

```diff
@@ -0,0 +1,173 @@
+
+        public Item(final long id) {
+            this.id = new SimpleObjectProperty<>(id);
+        }
+
+        public ReadOnlyObjectProperty<Long> idProperty() {
+            return id;
+        }
+    }
+
+    private static Parent createRoot() {
+        tableView = createTableView();
+        return tableView;
+    }
+
+    private static void addItem() {
+        items.add(0, new Item(counter.incrementAndGet()));
+
+        final TableViewSelectionModel<Item> sm = tableView.getSelectionModel();
+        if (sm.getSelectedItems().isEmpty()) {
+            sm.selectLast();
+        }
+        if (!tableView.isFocused()) {
+            tableView.requestFocus();
+        }
+    }
+
+    private static final AtomicBoolean itemsPopulatedAndCleared = new AtomicBoolean(false);
+    private static WeakReference<TableRow<Item>> firstRowRef;
+
+    private static TableView<Item> tableView;
+
+    private static TableView<Item> createTableView() {
+        final TableView<Item> tableView = new TableView<>(items);
+
+        tableView.setRowFactory(param -> {
+            final TableRow<Item> row = new TableRow<>();
+            if (itemsPopulatedAndCleared.get()) {
+                if (firstRowRef == null) {
+                    firstRowRef = new WeakReference<>(row);
+                }
+            }
+            return row;
+        });
+
+        final TableColumn<Item, Long> idColumn = new TableColumn<>();
+        idColumn.setCellValueFactory(cd -> cd.getValue().idProperty());
+
+        tableView.getColumns().add(idColumn);
+        return tableView;
+    }
+
+    @BeforeClass
+    public static void initFX() {
+        startupLatch = new CountDownLatch(1);
+        new Thread(() -> Application.launch(TestApp.class, (String[]) null)).start();
+        try {
+            if (!startupLatch.await(10, SECONDS)) {
+                fail("Timeout waiting for FX runtime to start");
+            }
+        } catch (final InterruptedException ex) {
+            fail("Unexpected exception: " + ex);
+        }
+    }
+
+    @AfterClass
+    public static void teardown() {
+        Platform.exit();
+    }
+
+    @Test
+    public void test_JDK8203345() throws Exception {
+        final boolean screenReaderPresent = screenReaderLatch.await(5_000, MILLISECONDS);
+        assumeTrue(screenReaderPresent);
+
+        runAndWait(() -> addItem());
+        runAndWait(() -> items.clear());
+
+        itemsPopulatedAndCleared.set(true);
+
+        for (int i = 0; i < 20; i++) {
+            runAndWait(() -> addItem());
+        }
+        runAndWait(() -> items.clear());
+
+        for (int j = 0; j < 5; ++j) {
+            System.gc();
+            System.runFinalization();
+            if (firstRowRef.get() == null) {
+                break;
+            }
+            MILLISECONDS.sleep(100);
+        }
+        assertEquals(null, firstRowRef.get());
+    }
+
+    private void runAndWait(final Runnable runnable) {
+        Util.runAndWait(runnable);
+    }
+}
+
```

Comment:
Since you need to change this file anyway (to add the copyright), can you also delete the extra blank line at the end of the file?